src/test/java/games/strategy/engine/data/annotations/ValidateAttachmentsTest.java

```diff
@@ -292,7 +292,7 @@ private static String validateAttachment(final Class<? extends IAttachment> claz
       // validate that there is a field and a getter
       Field field = null;
       try {
-        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, "m_" + propertyName, false);
+        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, propertyName, false);
```

Comment:
I looked at this once before when I first started working on TripleA.  I didn't modify it at the time because I wasn't sure where the requirement for the `m_` prefix came from.  I thought there must be some connection to the production code, but I couldn't find any at the time.

If this test class is not being run against any of our production attachments, then does anyone know what its purpose actually is?  It doesn't seem useful to test that a bunch of test-only annotations obey a test-only validation class.  Am I missing something?